<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serch Directory</string>
  </property>
  <widget class="QListWidget" name="listMessage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="container" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>avformat_find_stream_info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearchText">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search -R text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearchPlist">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search plist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>*.cc   *.h</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.plist</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>32</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>In Dir :     home   /</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDir">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>electron11/src</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Preferences</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnSearchText</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doFindText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSearchPlist</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>doFindPlist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>616</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doFindText()</slot>
  <slot>doFindPlist()</slot>
 </slots>
</ui>
